### DISCUSSION TITLE
What year was the first successful navigable submarine built? 🤿🚢

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1274] [Galaxy Trivia] [answered]
**Bonus:** Which country transferred ownership of the Panama Canal to Panama in 1999? 🇵🇦

**Comment by tomerf** [comment, marked as answer]
Cornelis Drebbel built the first navigatable submarine in 1620.

Bonus: The US controlled the Panama Canal until 1977, then it was in a joint control by both countries until it was taken over by Panama in 1999.

**Comment by peterbaccaro** [comment]
The first successful navigable submarine was built by Cornelis Drebbel in **1620** while working for the English Royal Navy.

On December 31, 1999, the **United States** officially handed the Panama Canal over to Panama's government, ending a long saga that had started a century and a half earlier.
